### PR TITLE
Add new INSTALLING.md

### PR DESCRIPTION
**Pull request by rishabhtanwar29**
### Description
INSTALLING.md consists of installation steps for ubuntu distribution
which we are going to bundle with our release artifacts.

Signed-off-by: Rishabh Tanwar <ritanwar@amazon.com>

### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: INSTALLING.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+#### Initiate data with specific HBA configuration
+
+For production or isolated environments, it is recommended to specify the allowed IP addresses that can access the system and a secure authentication method such as `md5` in the `pg_hba.conf`. For other supported methods see [Authentication Methods documentation](https://www.postgresql.org/docs/14/auth-methods.html).
+
+```sh
+${BABELFISH_HOME}/bin/initdb -D ${BABELFISH_DATA}/ -E "UTF8"
+```
+
+Once the data directory is initialized, edit the `${BABELFISH_DATA}/pg_hba.conf` file following the [vanilla Postgres HBA configuration](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html).
```
Comment (kuntalghosh) [inline INSTALLING.md]: Shouldn't this be https://www.postgresql.org/docs/13/auth-pg-hba-conf.html

File: INSTALLING.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+If you would like to create a local cluster for testing purposes, you can configure trust authentication when initializing the database to simplify authentication. The `--auth-host=trust` flag will create the cluster using trust authentication, and should not be included if you are creating an instance that will contain sensitive information:
+
+```sh
+${BABELFISH_HOME}/bin/initdb -D ${BABELFISH_DATA}/ -E "UTF8" --auth=trust --auth-host=trust --auth-local=trust
+```
+
+#### Initiate data with specific HBA configuration
+
+For production or isolated environments, it is recommended to specify the allowed IP addresses that can access the system and a secure authentication method such as `md5` in the `pg_hba.conf`. For other supported methods see [Authentication Methods documentation](https://www.postgresql.org/docs/14/auth-methods.html).
```
Comment (kuntalghosh) [inline INSTALLING.md]: https://www.postgresql.org/docs/13/auth-methods.html

File: INSTALLING.md (hunk 1 above)
Comment (rishabhtanwar29) [reply INSTALLING.md]: Updated!

File: INSTALLING.md (hunk 2 above)
Comment (rishabhtanwar29) [reply INSTALLING.md]: Updated!

**Review by kuntalghosh — APPROVED** [review]

**Pull request merged**